test/util/report.js

```diff
@@ -76,7 +76,7 @@ const getKeyboardRowName = function ($, $tableRow) {
  * @param {jQuery object} $tableRow - root example directory
  */
 const getAttributeRowName = function ($, $tableRow) {
-  // use the containt 'th' text to identify the row. If there is no text
+  // use the contents of the 'th' text to identify the row. If there is no text
```

Comment:
Wasn't sure about this one

Reply:
Thanks! I took your suggestion, but applied it manually since I was rebasing the branch for the `progid` change